bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/library/types/HSBType.java

```diff
@@ -269,7 +267,12 @@ public State as(Class<? extends State> target) {
         } else if (target == PercentType.class) {
             return new PercentType(getBrightness().toBigDecimal());
         } else {
-            return StateConverterUtil.defaultConversion(this, target);
+            // if the target is the same type return, otherwise give up
```

Comment:
In `DecimalType` you mean? Because DecimalType is the super class of `PercentType`, which is the super class of `HSBType`. So the protected method in `DecimalType` can delegate it to state?